src/CommunityToolkit.Maui.Core/Handlers/Popup/PopupViewHandler.shared.cs

```diff
@@ -49,4 +49,78 @@ public PopupViewHandler()
 		: base(PopUpMapper, PopUpCommandMapper)
 	{
 	}
-}
\ No newline at end of file
+}
+
+#if !(IOS || ANDROID || MACCATALYST || WINDOWS)
```

Comment:
@brminnick why not use a separated file?

Reply:
We will! 

I opened Discussion #301 to chat about the best/recommended way to do it.

For this PR, we'll keep with the current convention of `#if !(IOS || ANDROID || MACCATALYST || WINDOWS)` like we used in `Snackbar` and `Toast`. 

Then we'll move this into a separate file in the future once we've hammered out the best way to go about it in #301.

Long story short, I don't want that Discussion + future Proposal holding up this PR.